### ISSUE TITLE
Account management screen is displayed empty (just left tabs but empty content) when using Firefox in 

### ISSUE THREAD

**Issue by robotiko** [Strider-CD/strider#717]
Works correctly on Chrome, but on Firefox 35 it is shown empty.

**Comment by knownasilya** [comment]
Confirmed, I'll see what I can do later today.

**Comment by laurentj** [comment]
I confirm it. v1.6.6

Errors in the console : 

```
Empty string has been given to getElementById    app.js: 40188
```

Source code:

```javascript

			// Match html or make sure no context is specified for #id
			if ( match && (match[1] || !context) ) {

				// HANDLE: $(html) -> $(array)
				if ( match[1] ) {
					context = context instanceof jQuery ? context[0] : context;

					// scripts is true for back-compat
					// Intentionally let the error be thrown if parseHTML is not present
					jQuery.merge( this, jQuery.parseHTML(
						match[1],
						context && context.nodeType ? context.ownerDocument || context : document,
						true
					) );

					// HANDLE: $(html, props)
					if ( rsingleTag.test( match[1] ) && jQuery.isPlainObject( context ) ) {
						for ( match in context ) {
							// Properties of context are called as methods if possible
							if ( jQuery.isFunction( this[ match ] ) ) {
								this[ match ]( context[ match ] );

							// ...and otherwise set as attributes
							} else {
								this.attr( match, context[ match ] );
							}
						}
					}

					return this;

				// HANDLE: $(#id)
				} else {
// ========= LINE 40188  ----v
					elem = document.getElementById( match[2] ); 

					// Check parentNode to catch when Blackberry 4.6 returns
					// nodes that are no longer in the document #6963
					if ( elem && elem.parentNode ) {
						// Handle the case where IE and Opera return items
						// by name instead of ID
						if ( elem.id !== match[2] ) {
							return rootjQuery.find( selector );
						}

						// Otherwise, we inject the element directly into the jQuery object
						this.length = 1;
						this[0] = elem;
					}
```

I can't configure plugins !!!

**Comment by laurentj** [comment]
When we click on a tab, jQuery.fn.init() is called several times, and for one of these calls, the 'selector' argument is equals to "#", so match = [ "#", undefined, ""] and then document.getElementById(match[2]) fails.

**Comment by laurentj** [comment]
I've the same issue on a project configuration page. Impossible to modify setting of a project : page is empty. we only see tabs.

Please fix it in a 1.6.x, not in 1.7.0

**Comment by laurentj** [comment]
Sorry, the issue on a project configuration page is an other js error, and appears both on Firefox and Chrome, see #758
